### ISSUE TITLE
$expand not showing on single entity paths

### ISSUE THREAD

**Issue by bigred8982** [rbeauchamp/Swashbuckle.OData#55]
The $expand option is not showing up on paths created to query a single entity from an EntitySet. It looks like it is being added in the ODataSwaggerUtilities.CreateSwaggerPathForEntity() method. But for some reason it seems to not be part of the PathItem after all is said and done.

https://github.com/rbeauchamp/Swashbuckle.OData/blob/master/Swashbuckle.OData/Descriptions/ODataSwaggerUtilities.cs

I'm looking specifically at this code:

```C#
    get = new Operation()
    .Summary("Get entity from " + entitySet.Name + " by key.")
    .OperationId(entitySet.Name + "_GetById")
    .Description("Returns the entity with the key from " + entitySet.Name)
    .Tags(entitySet.Name)
    .Parameters(keyParameters.DeepClone()
    .Parameter("$expand", "query", "Expands related entities inline.", "string", false))
    .Parameters(keyParameters.DeepClone()
    .Parameter("$select", "query", "Selects which properties to include in the response.", "string", false))
    .Responses(new Dictionary<string, Response>().Response("200", "EntitySet " + entitySet.Name, entitySet.GetEntityType()).DefaultErrorResponse()),
```

I am using the sample application to test this.

**Comment by bigred8982** [comment]
Figured it out. Created a pull request for fix.

https://github.com/rbeauchamp/Swashbuckle.OData/pull/56

**Comment by rbeauchamp-lendingtree** [comment]
@bigred8982 Thanks for the pull request! Your fix is now available in v2.11.1
